vnext/ReactUWP/Modules/AppThemeModuleUwp.cpp

```diff
@@ -31,18 +31,48 @@ AppTheme::AppTheme(
     : react::windows::AppTheme(),
       m_wkReactInstance(reactInstance),
       m_queueThread(defaultQueueThread) {
+  try {
+    updateAndSubscribeForChanges();
+  } catch (winrt::hresult_error const &ex) {
+    if (ex.code() != 0x80070490) {
+      // PEER_E_NOT_FOUND Expected when app starts in the background.
+      winrt::throw_last_error();
+    }
+
+    m_leavingBackgroundRevoker =
```

Comment:
>m_leavingBackgroundRevoker [](start = 4, length = 26)

I don't know too much about background task. Is it possible to leave foreground and enter into background?